src/main/java/htsjdk/samtools/DuplicateScoringStrategy.java

```diff
@@ -80,6 +81,8 @@ public static short computeDuplicateScore(final SAMRecord record, final ScoringS
                         score += SAMUtils.getMateCigar(record).getReferenceLength();
                     }
                     break;
+                case RANDOM:
+                    score += (short) Math.sqrt(record.getReadName().hashCode());
```

Comment:
I can make that change if you'd like.  I felt a bit funny about it because the `enum` would have to have an instance of Murmur3, which felt funny to me.  But other than for the seed, `Murmur3` is stateless so it would work.  Thoughts?